app/Controllers/UserActionsController.php

```
@@ -104,76 +116,272 @@ public function reservationsPage(Request $request, Response $response, mysqli $d
+
+            $db->commit();
+
+            return $response->withHeader('Location', RouteTranslator::route('reservations') . '?updated=1')->withStatus(302);
+
+        } catch (\Throwable $e) {
+            $db->rollback();
+            error_log("Change reservation date error: " . $e->getMessage());
+            return $response->withHeader('Location', RouteTranslator::route('reservations') . '?error=db')->withStatus(302);
+        }
     }
     public function reservationsCount(Request $request, Response $response, mysqli $db): Response
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)